### PR TITLE
[OPENTOK-46753] suppress unnecessary warnings

### PR DESCRIPTION
**Pull request by jvm7293**

### CODE REVIEW

**Comment by jvm7293** [comment]
Remove unnecessary warnings specified in OPNETOK_46753
These changes are based on the commit 84d48e671779a46fbc3e49c2b130d45fd3b41d08 (which is currently in production)

File: mod_wsbridge/mod_wsbridge.c [hunk 1]
```diff
@@ -855,6 +855,10 @@ static switch_status_t channel_on_init(switch_core_session_t *session)
 	assert(channel != NULL);
 	switch_set_flag_locked(tech_pvt, TFLAG_IO);
 
+	switch_mutex_lock(globals.mutex);
```
Comment (odivorra) [inline mod_wsbridge/mod_wsbridge.c]: keep this as it was

**Review by brunorib — APPROVED** [review]
lgtm

**Comment by odivorra** [comment]
abandoned

**Pull request closed without merging**